proxy/src/control/destination/mod.rs

```diff
@@ -84,20 +82,13 @@ pub struct Resolution<B> {
     /// reference has been dropped.
     _active: Arc<()>,
 
-    /// Map associating addresses with the `Store` for the watch on that
-    /// service's metric labels (as provided by the Destination service).
-    ///
-    /// This is used to update the `Labeled` middleware on those services
-    /// without requiring the service stack to be re-bound.
-    metric_labels: HashMap<SocketAddr, Store<Option<DstLabels>>>,
-
     /// Binds an update endpoint to a Service.
     bind: B,
 }
 
-/// .
+/// Metadata describing an endpoint.
 #[derive(Clone, Debug, Hash, Eq, PartialEq)]
-struct Metadata {
+pub struct Metadata {
```

Comment:
Are you sure we need to make this `pub`? I don't see any new users in this PR but maybe I'm overlooking something.